packages/ai-robots-txt/package.json

```diff
@@ -0,0 +1,54 @@
+{
+  "name": "@hono/ai-robots-txt",
+  "version": "0.1.0",
```

Comment:
The Changeset will bump the version number automatically. So this number should be `0.0.0`.